src/encoder.rs

```diff
@@ -254,7 +289,7 @@ impl<W: Write> Encoder<W> {
             Repetitions(repeat) => {
                 self.w.write_le(Extension::Application as u8)?;
                 self.w.write_le(11u8)?;
-                self.w.write(b"NETSCAPE2.0")?;
+                self.w.write_all(b"NETSCAPE2.0")?;
```

Comment:
I also found clippy hard error-ing on this which I assume should've been `write_all`

Reply:
That is correct, good find.